src/aavrutiDJun20/MethodsOfCharacter.java

```diff
@@ -0,0 +1,42 @@
+//Print count of Digits, Letters, UpperCase, LowerCase and Special Character in given input '1rRpd3F9#K(E'
+
+package aavrutiDJun20;
+
+public class MethodsOfCharacter {
+
+	void verifyMethodsOfCharacter(String input) {
+		int digitCnt = 0;
+		int letterCnt = 0;
+		int upperCaseCnt = 0;
+		int lowerCaseCnt = 0;
+		int specialChar = 0;
+		
+		for(int index=0;index<input.length();index++) {
+			char ch = input.charAt(index);
+			if(Character.isDigit(ch) == true) 
+				digitCnt++;
+			else if(Character.isUpperCase(ch) == true) {
+				upperCaseCnt++;
+			    letterCnt++;
+			}
+			else if(Character.isLowerCase(ch) == true) {
+				lowerCaseCnt++;			
+				letterCnt++;
```

Comment:
Think in line number 20 and 24 letterCnt this variable is not needed.

Reply:
Done.